pandas/core/window.py

```diff
@@ -121,30 +121,20 @@ def validate(self):
         ]:
             raise ValueError("closed must be 'right', 'left', 'both' or " "'neither'")
 
-    def _convert_freq(self):
```

Comment:
Just a cleanup opportunity as you were looking at this?

Reply:
Correct. This function is only used once in the base class and found an opportunity to make the code path more direct.